tests/test_pango.py

```diff
@@ -0,0 +1,124 @@
+    text = "hello"
+    size = 1
+    folder = os.path.abspath(os.path.join("media", "texts"))
+    if not os.path.exists(folder):
+        os.makedirs(folder)
+    filename = os.path.join(folder, "hello.svg")
+    a = PangoText(text, size=10)
+    pangoManim = os.path.join(folder, a.text2hash() + ".svg")
+    surface = cairocffi.SVGSurface(filename, width, height)
+    context = cairocffi.Context(surface)
+    context.move_to(START_X, START_Y)
+    layout = pangocairocffi.create_layout(context)
+    layout.set_width(pangocffi.units_from_double(width))
+    fontdesc = pangocffi.FontDescription()
+    fontdesc.set_size(pangocffi.units_from_double(size * 10))
+    layout.set_font_description(fontdesc)
+    layout.set_text(text)
+    pangocairocffi.show_layout(context, layout)
+    surface.finish()
+    b = SVGMobject(filename)
+    assert len(a.submobjects) == len(b.submobjects)
+
+
+def test_rtl_text_text_svgobject():
+    """Checks number of submobjects generated when directly called using ``SVGMobject``"""
+    size = 1
+    text = rtl_text
+    folder = os.path.abspath(os.path.join("media", "texts"))
+    if not os.path.exists(folder):
+        os.makedirs(folder)
+    filename = os.path.join(folder, "hello.svg")
+    a = PangoText(text, size=1)
+    pangoManim = os.path.join(folder, a.text2hash() + ".svg")
+    surface = cairocffi.SVGSurface(filename, width, height)
+    context = cairocffi.Context(surface)
+    context.move_to(START_X, START_Y)
+    layout = pangocairocffi.create_layout(context)
+    layout.set_width(pangocffi.units_from_double(width))
+    fontdesc = pangocffi.FontDescription()
+    fontdesc.set_size(pangocffi.units_from_double(size * 10))
+    layout.set_font_description(fontdesc)
+    layout.set_text(text)
+    pangocairocffi.show_layout(context, layout)
+    surface.finish()
+    b = SVGMobject(filename)
+    assert len(a.submobjects) == len(b.submobjects)
+
+
+def test_font_face():
+    """Checks font face using submobject len"""
+    size = 1
+    text = rtl_text
+    font_face = "sans"
+    folder = os.path.abspath(os.path.join("media", "texts"))
+    if not os.path.exists(folder):
+        os.makedirs(folder)
+    filename = os.path.join(folder, "hello.svg")
+    a = PangoText(text, size=1, font=font_face)
+    pangoManim = os.path.join(folder, a.text2hash() + ".svg")
+    surface = cairocffi.SVGSurface(filename, width, height)
+    context = cairocffi.Context(surface)
+    context.move_to(START_X, START_Y)
+    layout = pangocairocffi.create_layout(context)
+    layout.set_width(pangocffi.units_from_double(width))
+    fontdesc = pangocffi.FontDescription()
+    fontdesc.set_family(font_face)
+    fontdesc.set_size(pangocffi.units_from_double(size * 10))
+    layout.set_font_description(fontdesc)
+    layout.set_text(text)
+    pangocairocffi.show_layout(context, layout)
+    surface.finish()
+    b = SVGMobject(filename)
+    assert len(a.submobjects) == len(b.submobjects)
```

Comment:
Question: How does this test the font face?

Reply:
AFAIK, if Pango uses the default font face for Arabic texts the SVG would not be formatted properly. SO checking for submobject between the one generated directly from Pango and also from manim would do and is a very basic simple check. Also, what are other properties of `mobjects` that can be compared?

Reply:
Not sure I follow. What I'm saying is that the two texts may have the same length of submobjects but have different font face. I'm not sure how to compare the font faces...

Reply:
Here you go how Pango has if the font is not there.
![random](https://user-images.githubusercontent.com/49693820/94579494-85519b80-0296-11eb-9a86-71f89c639ab0.png)
While if we select those fonts it will have the same length of submobject(manim parses it so). I also don't know how to check this more exactly.

Reply:
Yeah I see. So this test is not really checking for the font. In that case, we should remove it.

One way to go about it is to use the same framework as for comparing video frames. But @huguesdevimeux said this won't work with fonts _because of reasons_.